samples/router/src/hooks/useCustomRouting.ts

```diff
@@ -0,0 +1,36 @@
+import { useEffect } from "react"
+import useSendbirdStateContext from "@sendbird/uikit-react/useSendbirdStateContext"
+import { GroupChannelHandler } from '@sendbird/chat/groupChannel'
+import { useNavigate } from 'react-router-dom'
+
+
+const simpleUUID = () => {
+  return Date.now().toString(36) + Math.random().toString(36).substr(2)
+}
```

Comment:
nit: we have an `uuid` util function

Reply:
We cannot really use this for the sample ~ customers  should be able to use this as "self contained"

Reply:
Ah this is to show how to implement it, I got it